web/src/app/users/UserDetails.js

```diff
@@ -79,25 +82,24 @@ export default function UserDetails(props) {
 
   const user = _.get(data, 'user')
   const svcCount = serviceCount(user.onCallSteps)
+  const dialActions = [
+    {
+      label: 'Add Contact Method',
+      icon: <SettingsPhone />,
+      onClick: () => setCreateCM(true),
+    },
+  ]
+  if (user.contactMethods.length)
+    dialActions.push({
```

Comment:
Let's disable this instead of hiding, so what the user sees is consistent.

When a new user goes to the profile for the first time, they won't get an indication that the speed dial options changed in the background.